Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "northstar-discord-rpc"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+rrplug = { git = "https://github.com/catornot/rrplug.git", features=["presence"] }
```

Comment:
Pin rrplug, please.

Reply:
like to a commit? or actual version? 
(I will release rrplug v2 either this weekend or when 1.13 releases)

Reply:
Version is usually the way to go. I'd just release rrplug v2 now already tbh. Plan is to release 1.13 before 24th anyway, and that's like 2-3 days from now ^^